src/components/ui/BrowsePage/DataToolLinks.tsx

```diff
@@ -0,0 +1,117 @@
+import React from 'react';
+import { Button, ButtonGroup, Typography } from '@material-tailwind/react';
+import { Link } from 'react-router';
+
+import neuroglancer_logo from '@/assets/neuroglancer.png';
+import validator_logo from '@/assets/ome-ngff-validator.png';
+import volE_logo from '@/assets/aics_website-3d-cell-viewer.png';
+import copy_logo from '@/assets/copy-link-64.png';
+import type { OpenWithToolUrls } from '@/hooks/useZarrMetadata';
+import { copyToClipboard } from '@/utils/copyText';
+import FgTooltip from '../widgets/FgTooltip';
+
+export default function DataToolLinks({
+  title,
+  urls
+}: {
+  title: string;
+  urls: OpenWithToolUrls;
+}): React.ReactNode {
+  const [showCopiedTooltip, setShowCopiedTooltip] = React.useState(false);
+
+  const handleCopyUrl = async () => {
+    if (urls?.copy) {
+      await copyToClipboard(urls.copy);
+      setShowCopiedTooltip(true);
+      setTimeout(() => {
+        setShowCopiedTooltip(false);
+      }, 1000);
```

Comment:
Usually when I set these alerts to one second, Konrad asks me to extend the timeout. Maybe 2 - 3 seconds.